clients-mapreduce/src/test/java/uk/gov/gchq/palisade/clients/mapreduce/PalisadeInputSplitTest.java

```diff
@@ -44,7 +43,7 @@ public void shouldntAcceptNegativeLength() throws IOException {
         Assert.fail("exception expected");
     }
 
-    private StubResource stubResource = new StubResource("test type", "test id", "test format");
+//    private StubResource stubResource = new StubResource("test type", "test id", "test format");
```

Comment:
Why are the lines around StubResource commented out? either remove the lines completely or add comments to the code to explain